[DOCS] Fix typo (#67576)
elastic/elasticsearch: docs/reference/docs/multi-termvectors.asciidoc

You can specify existing documents by index and ID or 
provide artificial documents in the body of the request.  

> The index can be specified in the body of the request or in the request URI.

Reviewer: ```suggestion
You can specify the index in the request body or request URI.
```
